### PR TITLE
update

### PR DESCRIPTION
**Pull request by linuxyyds666**
Hello, I'm a newcomer. Recently I found that I couldn't open kernelsu debug mode when compiling the kernel, and I tried modifying the script according to the new bing's prompt. It  looks working fine.

### CODE REVIEW

File: kernel/setup.sh [hunk 1]
```diff
@@ -44,5 +44,5 @@ echo '[+] Add kernel su driver to Makefile'
 
 DRIVER_MAKEFILE=$DRIVER_DIR/Makefile
 grep -q "kernelsu" "$DRIVER_MAKEFILE" || printf "\nobj-y += kernelsu/\n" >> "$DRIVER_MAKEFILE"
-
+grep -q "kernelsu" drivers/Kconfig || sed -i '/endmenu/i source "drivers/kernelsu/Kconfig"' drivers/Kconfig
```
Comment (tiann) [inline kernel/setup.sh:47]: Please use $DRIVER_DIR instead of hardcode it

**Review by tiann — CHANGES_REQUESTED** [review]

**Comment by tiann** [comment]
By the way, you can add `ccflags-y += -DCONFIG_KSU_DEBUG` to Makefile to enable debug mode if you don't want to change config.

**Comment by tiann** [comment]
this means using "$DRIVER_DIR" instead, not $(DRIVER_DIR) or $DRIVER_DIR, you can ask new bing or chatgpt

**Comment by linuxyyds666** [comment]
> this means using "$DRIVER_DIR" instead, not $(DRIVER_DIR) or $DRIVER_DIR, you can ask new bing or chatgpt

OK.

**Review by tiann — CHANGES_REQUESTED** [review]

**Pull request closed without merging**